README.md

```diff
@@ -103,6 +103,224 @@ PSPDFKit.present('document.pdf', {
+
+  ```java
+package com.yourapp;
+
+import android.app.Application;
+import android.util.Log;
+
+import com.facebook.react.ReactApplication;
+import com.facebook.react.ReactInstanceManager;
+import com.facebook.react.ReactNativeHost;
+import com.facebook.react.ReactPackage;
+import com.facebook.react.shell.MainReactPackage;
+import com.react.pspdfkit.PSPDFKitPackage; // <--- import
+
+import java.util.Arrays;
+import java.util.List;
+
+public class MainApplication extends Application implements ReactApplication {
+
+  private final ReactNativeHost mReactNativeHost = new ReactNativeHost(this) {
+    @Override
+    protected boolean getUseDeveloperSupport() {
+      return BuildConfig.DEBUG;
+    }
+    
+    @Override
+    protected List<ReactPackage> getPackages() {
+       return Arrays.<ReactPackage>asList(
+           new MainReactPackage(), // <------- add comma
+           new PSPDFKitPackage()   // <------- add package
+        );
+    }
+  };
+  .....
+}
+  ```
+     
+12. Set default PSPDFKit theme. In `YourApp/android/app/src/main/res/values/style.xml` replace
+  ```xml    
+<style name="AppTheme" parent="Theme.AppCompat.Light.NoActionBar">
+  ```
+with
+  ```xml    
+<style name="AppTheme" parent="PSPDFCatalog.Theme.Light"> 
+  ```     
+13. Replace the default component from `index.android.js` with a simple touch area to present the bundled PDF:
+
+  ```javascript
+import React, { Component } from 'react';
+import {
+  AppRegistry,
+  StyleSheet,
+  NativeModules,
+  Text,
+  TouchableHighlight,
+  View
+} from 'react-native';
+
+var PSPDFKit = NativeModules.PSPDFKit;
+
+const DOCUMENT = "document.pdf";
+const LICENSE = "LICENSE_KEY_GOES_HERE";
+const CONFIGURATION = {
+  startPage : 3,
+  scrollContinuously : false,
+  showPageNumberOverlay : true,
+  grayScale : true,
+  showPageLabels : false,
+  pageScrollDirection : "vertical"
+};
+
+// Change 'YourApp' to your app's name.
+class YourApp extends Component {
+  _onPressButton() {
+    PSPDFKit.presentLocal(DOCUMENT, LICENSE, CONFIGURATION);
+  }
+
+  render() {
+    return (
+      <View style={styles.container}>
+      <Text style={styles.version}>{PSPDFKit.VERSION}</Text>
+        <TouchableHighlight onPress={this._onPressButton}>
+          <Text style={styles.text}>Tap to Open Document</Text>
+        </TouchableHighlight>
+      </View>
+    );
+  }
+}
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    justifyContent: 'center',
+    alignItems: 'center',
+    backgroundColor: '#F5FCFF',
+  },
+  text: {
+    fontSize: 20,
+    textAlign: 'center',
+    margin: 10,
+  }
+});
+
+// Change both 'YourApp's to your app's name.
+AppRegistry.registerComponent('YourApp', () => YourApp);
+  
+  ```  
+
+Your app is now ready to launch. From `YourApp` directory run `react-native run-android`.
+
+#### Configuration
+
+To copy a pdf document to your local device storage:
```

Comment:
Theres no filesystem path to files inside the assets. The only way to access them is using the `context.getAssets()` method. So that issue you mentioned will likely not be solved anytime soon.

Let me further explain what I mean: While one use case is to open a PDF file from the local file system (for example a downloaded file) another common scenario is to ship PDF documents with the `apk` file. On plain Android you would put that file into `src/main/assets/document.pdf` to it is packaged inside the `apk` file. This should work also for react, no matter how you then access the file at runtime.

PSPDFKit supports reading assets using the special `file:///android_asset/document.pdf` URI format. Internally we create an `AssetDataProvider` and load the file using that (not via the filesystem).

Since it's not a use case (but for development) to push a PDF document using `adb` to the device (no user will do this) we should probably also show how to open a document from the assets.